include/curl/curl.h

```diff
@@ -175,6 +175,16 @@ typedef enum {
 #define CURLSSLBACKEND_CYASSL CURLSSLBACKEND_WOLFSSL
 #define CURLSSLBACKEND_DARWINSSL CURLSSLBACKEND_SECURETRANSPORT
 
+/* bits for the CURLOPT_FOLLOWLOCATION option */
+#define CURLFOLLOW_ENABLE    1L /* generic follow redirects */
+
+/* Do not use the custom method in the follow-up request if the HTTP code
+   instructs so (301, 302, 303). */
+#define CURLFOLLOW_OBEYCODE  2L
```

Comment:
```suggestion
#define CURLFOLLOW_ENABLE     1L /* generic follow redirects */

/* Do not use the custom method in the follow-up request if the HTTP code
   instructs so (301, 302, 303). */
#define CURLFOLLOW_OBEYCODE   2L
```
nit: align these vertically to 3L?

Reply:
yes thanks, but I rather moved the 3 to the same column as 1 and 2